aip/client-libraries/4222.md

```diff
@@ -154,6 +160,46 @@ If none of the routing parameters matched their respective fields, the routing h
 Much like URL parameters, if there is more than one key-value pair to be sent, the `&`
 character is used as the separator.
 
+### `path_template` syntax
+
+As seen in the above examples, the `path_template` can use a variety of symbols
+that are interpreted by code generators during conversion to regular expressions
+or non-regular expression matcher implementations. The syntax for these symbols
+is as follows:
```

Comment:
"The `path_template` consists of segments delimited by the segment delimiter. " should be somewhere in here.

Reply:
Added.